### DISCUSSION TITLE
[Thank you] I think I talk for a lot of users/forkers when I say...

### DISCUSSION THREAD

**Discussion by Spacellary** [nikhilbadyal/docker-py-revanced#171] [General]
@nikhilbadyal Thank you so much for this project. The amount of convenience and time saved from this is underappreciated.

You're the best my friend.

Ah yea, btw, is mMicroG patching supported already? (The custom icons/etc)

**Comment by nikhilbadyal** [comment]
Sadly no, we still do not allow patching mMicrog. 
It anyway is non-interactive(in general case) and most of the time it isn't even visible on the launcher. I'm not sure if it's worth the efforts.

**Reply by Spacellary** [reply]
I agree, was just curious

**Comment by nikhilbadyal** [comment]
Closing as microg is there now.
